webpack/redux/reducers/RedHatRepositories/repositorySetRepositories.js

```diff
@@ -12,9 +13,23 @@ import {
 
 import { normalizeContentSetRepositories } from '../../actions/RedHatRepositories/repositorySetRepositories';
 
+const normalizeArch = (arch) => {
+  if (arch === 'noarch') {
+    return undefined;
+  }
+  return arch;
+};
+
+const substitutionMatches = (value1, value2) => {
+  if (_.isEmpty(value1) && _.isEmpty(value2)) {
```

Comment:
Cant you do 
```((_.isEmpty(value1) && _.isEmpty(value2)) || value1 == value2 ```

Reply:
i kinda like the more explicitness of what i have